server/src/domain/search/search.service.ts

```diff
@@ -160,7 +160,7 @@ export class SearchService {
   }
 
   async searchPerson(authUser: AuthUserDto, dto: SearchPeopleDto): Promise<PersonResponseDto[]> {
-    return await this.personRepository.getByName(authUser.id, dto.name);
+    return await this.personRepository.getByName(authUser.id, dto.name, dto.withHidden || false);
```

Comment:
After the default argument for the function is added, you can omit the `||false`

```suggestion
    return await this.personRepository.getByName(authUser.id, dto.name, dto.withHidden);
```